src/components/pill-selector-dropdown/PillSelectorDropdownField.js

```diff
@@ -43,10 +45,12 @@ type State = {
 
 class PillSelectorDropdownField extends React.PureComponent<Props, State> {
     static defaultProps = {
+        dropdownFilter: defaultDropdownFilter,
```

Comment:
Ah yes, I thought I had updated this. Probably just set `defaultDropdownRenderer` and forgot about this one.